cogs/source/source.py

```diff
@@ -12,15 +21,22 @@ async def postSourceFunction(ctx, imageURL):
         anilistID = f"{result.docs[0].anilist_id}"
         episode = f"{result.docs[0].episode}"
         similarity = float(result.docs[0].similarity)
```

Comment:
Have you thought about doing something along the lines of:
```py
        anilistID = f"{result.docs[0].anilist_id}" or "No Anilist ID Found"
```
since `None or "No Anilist ID Found"` resolves to `"No Anilist ID Found"`?

Reply:
Changed to this method. I didn't realise you could do something like this, sorry